samples/msal-node-samples/standalone-samples/README.md

```diff
@@ -20,23 +24,40 @@ HTTP:
+You can also download the repository as a zip file by selecting "Download ZIP" from the root repository's dropdown "Code" menu. Once you've downloaded the ZIP file, you can decompress it locally and explore the code.
 
 ### Pre-requisites
-- Ensure [all pre-requisites](../../../lib/msal-node/README.md#prerequisites) have been completed to run msal-node.
-- Install Node.js if needed (https://nodejs.org/en/).
-- Build the `msal-node` project with instructions provided in the [`README.md`](../../../lib/msal-node/README.md) or using the command `npm run build:package`.
+- By using MSAL Node, you are working with the Microsoft Identity ecosystem.  Read about [App Registration](https://docs.microsoft.com/en-us/graph/auth-register-app-v2) and register one for use with this code.
+- Install [Node.js](https://nodejs.org/en/) if needed
+- Install the MSAL Node package:  
+```bash
+npm install @azure/msal-node
+```
+- If you are customizing or building locally:
+```bash
+npm run build:package
+```
 
-Once you've completed all the pre-requisites, you can navigate to any of the sample application directories under `standalone-samples` and either execute each sample as-is, or customize it to use your own AzureAD app registration.
+Next, we navigate to the sample application directories under `standalone-samples` and either execute each sample as-is, or customize it to use your own [App Registration](https://docs.microsoft.com/en-us/graph/auth-register-app-v2).
 
-From the repository's root directory, navigate to a sample application (for example, the auth-code sample):
+From the repository's root directory, navigate to a sample application:
 
 ```bash
 $ cd samples/msal-node-samples/standalone-samples/auth-code
 ```
 
+### Which sample should I review?
+
+Review our [scenario docs](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-flows-app-scenarios) to pick a sample. 
+
+Continue to the appropriate sample to learn how it works, or to build it in to your existing app.
+
+- [Auth Code](auth-code/readme.md)  [web app]
+- [Client Credentials](client-credentials/readme.md) [console app]
+- [Device Code](device-code/readme.md)
+- [On Behalf Of](on-behalf-of/web-app/readme.md) [calling MS Graph]
+- [Silent Flow](silent-flow/readme.md)
+
 ### Configure the application
 
 Each application has an `index.js` file, which is considered the entrypoint of the application. This file both contains the MSAL configuration and the application code that shows how to use each authorization flow. The sample works out-of-the-box with the default configuration in the `index.js` files, but you can customize it to use your own AzureAD app registration by changing the configuration values to match your app registration's. Below you'll find an example of what the main MSAL configuration object looks like. This is the object you'll want to customize to match your app registration.
```

Comment:
Like we discussed, we may want to remove the "out-of-the-box" claim until we replace the client IDs with the ones for lab app registrations that any user can authenticate with.

Reply:
I agree, I will remove the out of the box claims.